modules/core/src/main/kotlin/com/metalbear/mirrord/MirrordApi.kt

```diff
@@ -217,7 +217,7 @@ class MirrordApi(private val service: MirrordProjectService, private val project
             return
         }
 
-        service.notifier.notification("Enjoying mirrord? Don't forget to leave a review! Also consider giving us some feedback, we'd highly appreciate it!", NotificationType.INFORMATION).withLink("Review", "https://plugins.jetbrains.com/plugin/19772-mirrord/reviews").withLink("Feedback", FEEDBACK_URL).withDontShowAgain(MirrordSettingsState.NotificationId.PLUGIN_REVIEW).fire()
```

Comment:
Lets remove the "Having some trouble" part, why spoil a nice moment

Reply:
So: 
> Enjoying mirrord? Don't forget to leave a review! And don't hesitate to ask for help on our Discord.

Or just:
> Enjoying mirrord? Don't forget to leave a review!

(and the `Get Help on Discord` button on the bottom of the notification)?

Reply:
Changed to github link.